py/desispec/scripts/proc.py

```diff
@@ -759,14 +770,14 @@ def main(args=None, comm=None):
                 cmd += ' -o {}'.format(framefile)
                 cmd += ' --psferr 0.01'
 
-                if args.gpuspecter:
-                    cmd += ' --gpu-specter'
+                if args.use_specter:
+                    cmd += ' --use-specter'
                     #- default for CPU is nsubbundles=6 but gpu_specter only allows 1, 5, or 25
                     cmd += ' --nsubbundles 5'
```

Comment:
Good point.  Since desi_extract_spectra now has a default of --nsubbundles=5 for both specter and gpu_specter, we don't have to explicitly set it here in desi_proc for either case.  I'll remove this.

Correcting previous misstatements I've made: at the library level, both `gpu_specter.core.extract_frame` and `specter.extract.ex2d` use nsubbundles=1 as default (i.e. no subbundles), not 5 and 6 as I had previously thought.  So to use subbundles we do have to set it somewhere, and the desi_extract_spectra options is that place (not desi_proc).